src/main/webapp/app/core/util/alert.service.ts

```diff
@@ -132,38 +148,37 @@ export class AlertService {
         const alertInternal = { ...alert } as AlertInternal;
         alertInternal.isOpen = true;
 
-        if (!alert.disableTranslation) {
+        if (!alert.disableTranslation && (alert.translationKey || alert.message)) {
+            // in case a translation key is defined, we use it to create the message
+            // Note: in most cases, our code passes the translation key as message
+            let translationKey: string;
             if (alert.translationKey) {
-                // in case a translation key is defined, we use it to create the message
-                const translatedMessage = this.translateService.instant(alert.translationKey, alert.translationParams);
-                // if translation key exists
-                if (translatedMessage !== `${translationNotFoundMessage}[${alert.translationKey}]`) {
-                    alertInternal.message = translatedMessage;
-                } else if (!alertInternal.message) {
-                    alertInternal.message = alert.translationKey;
-                }
-            } else if (alertInternal.message) {
-                // Note: in most cases, our code passes the translation key as message
-                alertInternal.message = this.translateService.instant(alertInternal.message, alert.translationParams);
+                translationKey = alert.translationKey;
+            } else {
+                translationKey = alert.message!;
```

Comment:
I don't quite get this one. Why would you want to set the message as translationKey?